src/Altinn.Platform/Altinn.Platform.Events/Tests/Data/apps/tests-sailor.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "tests-sailor",
+  "org": "tests",
+  "title": {
+    "nb": "Testapplikasjon",
+    "en": "Test application"
+  },
+  "process": {
+    "currentTask": {
+      "elementId": "Task_1",
+      "name": "FormFilling"
+    }
```

Comment:
Not quite sure what this file represents? 
Might be appmetadata, but then process should not be included in the file and the name is also incorrect?

Reply:
deleted